frontend/src/components/organisms/forms/sponsorshipForm.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { SubmitHandler, useForm } from "react-hook-form";
+import { Form, Input, Button, InputNumber, Upload} from "antd";
+import { UploadOutlined } from '@ant-design/icons';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ISPP-Eventia/eventus/pullRequest?prid=9091638)
```suggestion
import { UploadOutlined } from "@ant-design/icons";
```